recipes/snakelines/meta.yaml

```diff
@@ -1,30 +1,30 @@
 {% set name = "snakelines" %}
-{% set version = "1.1.4" %}
+{% set version = "1.1.8" %}
 
 package:
   name: {{ name }}
   version: {{ version }}
 
 source:
   url: https://github.com/jbudis/snakelines/archive/refs/heads/master.zip
-  sha256: b19d59a0491bbbb51df6af2b9a9c614df6c029904abc6f9faf75ec784b918013
+  sha256: 0fa213a35bf75ca559ddad6bf35a03ffb7b64ca3f5575f5efb0faa4223aab8bb
 
 build:
   number: 0
   noarch: generic
 
 requirements:
   run:
-    - python ==3.7.6
+    - python >=3.9,<3.10
     - tk ==8.6.10
```

Comment:
Probably a good idea to either not pin versions of all dependencies, or at least pin to a known-working range like for Python. I believe pinning in `meta.yml` should generally be used to avoid known-bad combinations rather than force the exact versions that worked at some point in time, as it makes it hard to have multiple packages installed in a single env. Pinning to ensure exact reproducibility is done at the environment level (by end users).

Reply:
Good point. We have specified our dependencies to a tested working range. We will update dependencies in next version of Snakelines, so these will probably change in the near future.